src/reducers/Jobs.js

```diff
@@ -47,13 +49,17 @@ const initialLabels = {
 
 const initialState = {
   fetchState: {inFlight: false},
+  runIds: [],
```

Comment:
i don't really like the term `run`. it tells me nothing.

Reply:
Ok ok